.prettierignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Do we need anything else in here? Like any of the package built files, package node modules, storybook build etc?

Reply:
Hmm, a good thought. Let me take a look in the AM.

Reply:
You know, now that I'm looking at it, we don't need this file at all with our current setup since they are all git ignored. Since our set up is:

1. Format on save
2. Format on precommit hook

Any files that are never edited or ignored by git won't be formatted with Prettier.